gramps_webapi/data/apispec.yaml

```diff
@@ -554,7 +554,30 @@ paths:
           description: "Not Found: No people found."
         422:
           description: "Unprocessable Entity: Invalid or bad parameter provided."
-
+    post:
+      tags:
+      - people
+      summary: "Add a new person to the database."
+      operationId: createPerson
+      security:
+        - Bearer: []
+      consumes:
+        - application/json
+      parameters:
+        - in: body
+          name: person
+          description: The person to add
+          schema:
+            $ref: "#/definitions/Person"
+      responses:
+        201:
+          description: Created
+        400:
+          description: "Bad Request: Malformed request could not be parsed."
+        401:
+          description: "Unauthorized: Missing authorization header."
+        422:
```

Comment:
When is this thrown?

Reply:
422 is currently not thrown at all, true. I left it in because we will need it in case we add some query parameters in the future (although currently I cannot think of any...)